### DISCUSSION TITLE
Can ui choose other languages other than English?

### DISCUSSION THREAD

**Discussion by zhangyaodan1379** [alagrede/znote-app#36] [Q&A]
Such as Chinese Japanese

**Comment by alagrede** [comment]
Hello @zhangyaodan1379,
That's right, I know it's confusing to have an app in a foreign language especially with another alphabet  😅
I think it could be done pretty easily but it still requires a significant workload to translate the software into all relevant languages.

Maybe I can plan this feature before the end of the year. 📝
I'll get back to you, when I will deploy the translations.

Regards,
Anthony

**Reply by alagrede** [reply]
Before it evolves, we have already made changes to allow writing in Chinese notes, titles...

**Reply by zhangyaodan1379** [reply]
Can you send me a tutorial? I have not received the email sent, and can other languages such as python run on the znote app?

**Reply by alagrede** [reply]
Hello @zhangyaodan1379, 

A better emails tutorials process will come soon. Currently, the best way to learn about new features and possible uses is to check the Znote blog https://blog.znote.io
Regarding the execution of languages, I am focusing on providing the best possible experience in JavaScript before allowing others languages. I don't have a release date for this yet. Before a better Python integration, you can still run your Python scripts via the exec command and play around with the results like this:
```
const { stdout, stderr } = await exec('python yourScript.py');
print(stdout);
```

Hope this helped you.
Anthony
